gordon/main.py

```diff
@@ -83,14 +83,33 @@ def setup(config_root=''):
     return config
 
 
+def _log_or_raise_exceptions(errors, debug):
+    log_level_func = logging.warn
+    if not debug:
+        log_level_func = logging.error
+
+    base_msg = 'Plugin "{name}" was not loaded:'
+    for name, exc in errors:
+        msg = base_msg.format(
+            name=name, etype=exc.__class__.__name__, emsg=exc)
```

Comment:
I don't thing the `etype` and `emsg` args are being used by `format` since base_msg doesn't include it.

Reply:
damnit, I reorganized and I missed that part. good catch!